index.js

```diff
@@ -15,20 +20,39 @@ const prettySyntaxError = (err) => {
   }
 };
 
+const warnIf = (condition, warning) => {
+  if (condition) {
+    logger.warn(`${warning}`);
+  }
+};
+
 class BabelCompiler {
   constructor(config) {
     if (!config) config = {};
     const options = config.plugins &&
```

Comment:
`const options = config.plugins.babel || config.plugins.ES6to5 || {};` while we at it